examples/transfer_sequence.ts

```diff
@@ -0,0 +1,45 @@
+import { alice, bob, charlie, dave, Rune, RunicArgs, Sr25519, ValueRune } from "capi"
+import { Balances, System } from "westend_dev/mod.ts"
+import { upperFirstCase } from "../deps/case.ts"
+
+await aliceFree()
+  .log(msg("Alice", "before"))
+  .chain(() =>
+    Object
+      .entries({ bob, charlie, dave })
+      .reduce<ValueRune<any, any>>((acc, args) =>
```

Comment:
This isn't the prettiest (both the `reduce` call, and the `ValueRune` type args. How shall we clean this up? Is a new Rune utility required?